<launch>
  <node pkg="dogsim" type="simulated_dog_position_detector" name="dog_position_detector" output="screen"/>
</launch>
